Build.md

```diff
@@ -2,100 +2,198 @@
 
 Follow these instructions if you want to get started developing on TrenchBroom.
 
-To see how releases of TB are packaged, consult our CI scripts instead.
+To see how releases of TrenchBroom are packaged, consult our CI scripts instead.
 
 ## All Platforms
 
 First, clone the TrenchBroom repository. If you are using the official repository and not a fork, you can clone the
 repository by running
 
-```
+```bash
 git clone --recursive https://github.com/kduske/TrenchBroom.git
 ```
 
+---
+
 ## Windows
 
-- Visual Studio 2017 or 2019 can be used for development. Code needs to compile with VS 2017 (that's what our releases are built with). 
-- For Visual Studio 2017:
-    - In the Visual Studio Installer, you'll need to install:
-      - Workloads 
-        - **Desktop development with C++**
-      - Individual components
-        - **VC++ 2017 version 15.9 v14.16 latest v141 tools**
```

Comment:
Just want to confirm what I had there "VC++ 2017 version 15.9 v14.16 latest v141 tools" is no longer an option in the VS2017 installer? I thought I copied the checkbox label verbatim, but maybe they changed the name of the option.